packages/bitcore-node/src/providers/chain-state/evm/types.ts

```diff
@@ -263,12 +262,11 @@ export interface EventLog<T> {
   blockNumber: number;
   raw?: { data: string; topics: any[] };
 }
-export interface ERC20Transfer
-  extends EventLog<{
+export type ERC20Transfer = EventLog<{
     [key: string]: string;
-  }> {}
+  }>
 
-  export interface IEVMCachedAddress {
+export interface IEVMCachedAddress {
     address: string;
     tokenAddress?: string;
   }
```

Comment:
There seems to be some issues with indentation at the end of this file that's being missed by eslint. I was able to fix it by adding `'@stylistic/indent': ['error', 2, { SwitchCase: 1 }],` to the lint config.